rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/Transaction.java

```diff
@@ -1025,4 +1045,14 @@ public boolean hasPendingActions() {
 
         return participants;
     }
+
+    public static final class LRAStatusReason {
+        public static final String TIMEOUT = "LRA timeout";
```

Comment:
Note that you may still include descriptive strings as fields of `Enum` datatypes so you can keep the reason text if you use an Enum data structure to model the reasons for each state transition.